lib/menu.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'send_sms'
+
+class Menu
+
+attr_reader :menu, :selection_of_dishes, :message
+    
+    def initialize
+        @menu = {
+            "lasagne" => 12,
+            "oysters" => 14,
+            "fries" => 4,
+            "green_salad" => 9,
+            "rump_steak" => 20,
+            "tiramisu" => 25
+        }
+        @selection_of_dishes = []
+        #@texter = SendText.new
+    end
+
+    def show_menu
+        puts @menu
+    end
+
+    def order(dish, quantity = 1) #
+        # For more interactivity, I could:
+        # puts "Which dish(es) would you like to order?"
+        # @selection_of_dishes = gets.chomp
+        puts "#{quantity} x #{dish}(s) added to order."
+        @selection_of_dishes << @menu.fetch(dish) * quantity
+    end
```

Comment:
Good implementation, I would consider also maybe checking that the dish would be a valid dish?

Reply:
Yeah, a really good point. I was aware that this relied heavily on good spelling, etc., on behalf of the user!